src/app/main/main.component.ts

```diff
@@ -289,48 +387,84 @@ export class MainComponent implements OnInit, OnDestroy {
   }
 
   // 'count' is used to track the number of request. Each request is 10MB.
-  getResults(count: number, projectId: string, inputPageToken: string, location: string, jobID: string): Promise<BigQueryResponse> {
+  getResults(count: number, projectID: string, inputPageToken: string, location: string, jobID: string): Promise<BigQueryResponse> {
     if (!inputPageToken || count >= MAX_PAGES) {
       // Force an update feature here since everything is done.
       this.rows = this.rows.slice(0);
       return;
     }
     count = count + 1;
-    return this.dataService.getResults(projectId, jobID, location, inputPageToken, this.columns, this.stats).then(({ rows, stats, pageToken }) => {
+    return this.dataService.getResults(projectID, jobID, location, inputPageToken, this.columns, this.stats).then(({ rows, stats, pageToken }) => {
       this.rows.push(...rows);
       this.stats = stats;
       this._changeDetectorRef.detectChanges();
-      return this.getResults(count, projectId, pageToken, location, jobID);
+      return this.getResults(count, projectID, pageToken, location, jobID);
     });
   }
 
+  convertToUserQuery(geovizQuery : string) : string {
+    if (!geovizQuery) return '';
+
+    var lines = geovizQuery.split('\n');
+    var userQueryStarted = false;
+    var userQuery = '';
```

Comment:
var --> const

Reply:
Done.

Reply:
still seeing two `vars` above, did this get pushed?

Reply:
Didn't know the diff between var and let. Done.